autofs-5.0.5 - fix restart

From: Ian Kent <raven@themaw.net>

The recent LSB improvement change has introduced a problem with
the restart action.
---

 redhat/autofs.init.in |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)


diff --git a/redhat/autofs.init.in b/redhat/autofs.init.in
index 1e926ce..e2a4b78 100644
--- a/redhat/autofs.init.in
+++ b/redhat/autofs.init.in
@@ -126,7 +126,7 @@ function stop() {
 }
 
 function restart() {
-	status > /dev/null 2>&1
+	status autofs > /dev/null 2>&1
 	if [ $? -eq 0 ]; then
 		stop
 	fi
